openqa-maintenance.py

```diff
@@ -19,14 +19,25 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+
+import os.path as opa
 import re
 import sys
 from datetime import date
 import md5
-import json
+
+try:
+    import simplejson as json
+except ImportError:
+    import json
+
+try:
+    from simplejson import JSONDecodeError
+except ImportError:
```

Comment:
and err, this is buggy we really need simplejson because json doens't support JSONDecodeError  before python 3.5